helm/oncall/templates/integrations/deployment.yaml

```diff
@@ -0,0 +1,107 @@
+{{- if .Values.detached_integrations.enabled -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "oncall.detached_integrations.fullname" . }}
+  labels:
+    {{- include "oncall.detached_integrations.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.detached_integrations.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "oncall.detached_integrations.selectorLabels" . | nindent 6 }}
+  strategy:
+    {{- toYaml .Values.detached_integrations.updateStrategy | nindent 4 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        random-annotation: {{ randAlphaNum 10 | lower }}
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "oncall.detached_integrations.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "oncall.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      initContainers:
+        {{- include "oncall.initContainer" . | indent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
```

Comment:
similar here, consider creating a helper function to generate the `image` value ([duplicated](https://github.com/search?q=repo%3Agrafana%2Foncall%20%7B%7B%20.Values.image.repository%20%7D%7D%3A%7B%7B%20.Values.image.tag%20%7C%20default%20.Chart.AppVersion%20%7D%7D&type=code) in 5 spots now)